graphdatascience/procedure_surface/arrow/wcc_arrow_endpoints.py

```diff
@@ -0,0 +1,236 @@
+            computation_result["computeMillis"],
+            computation_result["postProcessingMillis"],
+            0,
+            mutate_result.nodePropertiesWritten,
+            computation_result["configuration"],
+        )
+
+    def stats(
+        self,
+        G: Graph,
+        threshold: Optional[float] = None,
+        relationship_types: Optional[List[str]] = None,
+        node_labels: Optional[List[str]] = None,
+        sudo: Optional[bool] = None,
+        log_progress: Optional[bool] = None,
+        username: Optional[str] = None,
+        concurrency: Optional[int] = None,
+        job_id: Optional[str] = None,
+        seed_property: Optional[str] = None,
+        consecutive_ids: Optional[bool] = None,
+        relationship_weight_property: Optional[str] = None,
+    ) -> WccStatsResult:
+        config = self._build_configuration(
+            G,
+            concurrency,
+            consecutive_ids,
+            job_id,
+            log_progress,
+            None,
+            node_labels,
+            relationship_types,
+            relationship_weight_property,
+            seed_property,
+            sudo,
+            threshold,
+        )
+
+        job_id = JobClient.run_job_and_wait(self._arrow_client, WCC_ENDPOINT, config)
+        computation_result = JobClient.get_summary(self._arrow_client, job_id)
+
+        return WccStatsResult(
+            computation_result["componentCount"],
+            computation_result["componentDistribution"],
+            computation_result["preProcessingMillis"],
+            computation_result["computeMillis"],
+            computation_result["postProcessingMillis"],
+            computation_result["configuration"],
+        )
+
+    def stream(
+        self,
+        G: Graph,
+        min_component_size: Optional[int] = None,
+        threshold: Optional[float] = None,
+        relationship_types: Optional[List[str]] = None,
+        node_labels: Optional[List[str]] = None,
+        sudo: Optional[bool] = None,
+        log_progress: Optional[bool] = None,
+        username: Optional[str] = None,
+        concurrency: Optional[int] = None,
+        job_id: Optional[str] = None,
+        seed_property: Optional[str] = None,
+        consecutive_ids: Optional[bool] = None,
+        relationship_weight_property: Optional[str] = None,
+    ) -> DataFrame:
+        config = self._build_configuration(
+            G,
+            concurrency,
+            consecutive_ids,
+            job_id,
+            log_progress,
+            min_component_size,
+            node_labels,
+            relationship_types,
+            relationship_weight_property,
+            seed_property,
+            sudo,
+            threshold,
+        )
+
+        job_id = JobClient.run_job_and_wait(self._arrow_client, WCC_ENDPOINT, config)
+        return JobClient.stream_results(self._arrow_client, job_id)
+
+    def write(
+        self,
+        G: Graph,
+        write_property: str,
```

Comment:
TODO: respect that property
It seems that currently the Python client does not support this parameter in conjunction with sessions oO